src/components/common/tabs/Tabs.tsx

```
@@ -0,0 +1,30 @@
+import { TabProps } from '../../../types/public/donate-page/DonateTab';
+import './Tabs.scss';
+
+export const Tabs = ({ tabs, activeTab, setActiveTab }: TabProps<number>) => {
+    return (
+        <div className="tabsContainer">
+            {tabs.map(({ id, label, disabled }) => {
+                const isActive = activeTab === id;
+                const tabClass = `tab ${isActive ? 'active' : ''}`;
+                return disabled ? (
+                    <div className="tooltip-container top">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)